doc/ref_model/chapters/chapter08.md

```diff
@@ -98,8 +98,63 @@ The Cloud Services and the Cloud Resources Brokers provide value-added services
 
 <a name="8.2.4"></a>
 ### 8.2.4 Multi-Cloud Interactions Model
-<a name="8.2.4"></a>
-
+<a name="8.2.4.1"></a>
+### 8.2.4.1 Introduction
+To realise a federated cloud requires the definition and agreement on a set of APIs. These APIs should allow each of the parties to interact cooperatively and need to cover the management layer: business management and service operations interactions; as well as the data plane, customer and user, transactions and conversational interfaces.
+
+As outlined in "Figure 8-3 - Conceptual Architecture of a Telco Operator Platform" above the exposure point for the Management Interactions is the "Cloud Service Broker" and the "Cloud Resource Broker". The set of interactions that these interface points need to provide are defined by the "Multi-Cloud Interaction Model". This provides a taxonomy for the interactions and is illustrated below:
+
+<p align="center"><img src="../figures/rm-chap8-multi-cloud-interactions-02.png" alt="Multi-Cloud Interactions Model"></p>
```

Comment:
Please share the source of the figure in the [figures folder](https://github.com/cntt-n/CNTT/tree/master/doc/ref_model/figures).

Reply:
@CsatariGergely diagrams are here: https://github.com/jbhartley/CNTT/blob/master/doc/ref_model/artefacts/anuket-ref-model-chap-8-multicloud-interactions-diagrams-01.pptx